doc/DOWNLOADS.md

```diff
@@ -90,7 +90,7 @@ Portable packages are also available. They can be run from your home directory,
 - [Download Nicotine+ 64-bit Windows Portable Package](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/windows-x86_64-package.zip)  [[SHA256](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/windows-x86_64-package.zip.sha256)]
 - [Download Nicotine+ 32-bit Windows Portable Package](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/windows-i686-package.zip)  [[SHA256](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/windows-i686-package.zip.sha256)]
 
-### Scoop
+### [Scoop](https://scoop.sh/)
 
 Nicotine+ can be installed using [Scoop](https://scoop.sh/). Run the following:
```

Comment:
The link in the heading is not necessary, since there is already a link to the same URL in the body text.